cli/functionary/package.py

```diff
@@ -61,6 +62,11 @@ def create_cmd(ctx, language, name, output_directory):
     if not dir.exists():
         dir.mkdir()
 
+    elif os.listdir(dir):
+        raise click.ClickException(
+            f"Create command failed: {output_directory + '/' + name} is not empty."
```

Comment:
Let's have this say:

```suggestion
            f"{output_directory + '/' + name} is not empty. Destination must be a new or empty directory."
```